src/originalBudgetTrackingApp/export/exportTransactions.ts

```diff
@@ -13,8 +13,7 @@ export async function createTransactionsInExternalVendors(
   const executionResult = {};
   const allTransactions = _.flatten(Object.values(companyIdToTransactions));
 
-  for (let j = 0; j < outputVendors.length; j++) {
-    const outputVendor = outputVendors[j];
+  const exportPromises = outputVendors.map(async (outputVendor) => {
```

Comment:
Two points just about coding practices, this specific code bothers me from the first time ;-)

I don't like **loop** with **if** inside, I'm preferring `filter` and `map` in this case.